android/assets/jsons/Civ V - Vanilla/Buildings.json

```diff
@@ -937,7 +937,7 @@
 	{
 		"name": "Manhattan Project",
 		"isNationalWonder": true,
-		"uniques": ["Enables nuclear weapon", "Triggers a global alert upon completion"],
+		"uniques": ["Enables nuclear weapon", "Triggers a global alert upon [completion]"],
```

Comment:
I like the idea of templating this, but in practice it means an additional translation for "completion" - which will arrive without context, a separate notification, which means that anyway you have only the preprogrammed options...
In short, in this specific instance, templating adds complexity instead of removing it.

Reply:
Good point, I'll revert this change